### DISCUSSION TITLE
Does zitadel support passwordless authentication?

### DISCUSSION THREAD

**Discussion by vgtom** [zitadel/zitadel#2029] [General]
if there are any plans to support passwordless authentication, i would like to contribute code to it?
this is why we need a chat channel...have you considered telegram or mattermost?

**Comment by fforootd** [comment]
Hi @vgtom nice to hear

ZITADEL already supports passwordless authentication based on FIDO2, some minor improvements could be made in the login and register parts if you are interested.

Regarding the chat what are the things you want to discuss in a chat?
The big drawback we see with chats in general is that they generate a closed environment which is not really accessible.

**Reply by fforootd** [reply]
In regard to the chat part.

https://github.com/caos/zitadel/discussions/1919#discussioncomment-1012111

**Reply by vgtom** [reply]
ok thanks...

forget about the chat...but lets say you want to give me an intro to the zitadel codebase for me to start contributing...is it possible to have a screen shared video meeting?

**Reply by fforootd** [reply]
For sure, we would be happy to give you a walk through.

What time and date would be feasible for you? In the next 2 weeks a lot of people are in the holidays, but still I think it would be possible.

CC @livio-a

**Reply by vgtom** [reply]
Lets fix a time sometime next week...I can accommodate Zurich or CET timezone...Please let me know when ur free and can do it

**Reply by fforootd** [reply]
Great, I will come forward with a slot.

**Reply by fforootd** [reply]
@vgtom How about 21st July 11:00 CEST?

@livio-a and I could show you around the code and get you up to speed 😁

**Reply by vgtom** [reply]
yes 21st July 11:00 CEST should be fine although i need to get my covid vaccine in the morning...but i think i will make the meeting.

Can we also make a provision to record the meeting for future reference?

**Reply by fforootd** [reply]
We can change to the afternoon if it fits better for you.

I think we could use https://meet.init7.net/en/ to meet and record the appointment!

We internally rely on Google Workspace but do not have the subscription to record meetings 😁

**Reply by vgtom** [reply]
ok...lets change to the afternoon.

yeah the meeting link looks fine!!

**Reply by fforootd** [reply]
So would 14:00 be fine?

And let's use this link https://meet.lax.init7.net/zitadel-contrib

**Reply by vgtom** [reply]
yes 14:00 CEST should work for me!!
ok, the meeting link is fine...

looking forward to talking to you tomorrow!
